x/bank/client/rest/grpc_query_test.go

```diff
@@ -93,3 +95,73 @@ func (s *IntegrationTestSuite) TestTotalSupplyGRPCHandler() {
 		})
 	}
 }
+
+func (s *IntegrationTestSuite) TestBalancesGRPCHandler() {
+	val := s.network.Validators[0]
+	baseURL := val.APIAddress
+
+	testCases := []struct {
+		name     string
+		url      string
+		headers  map[string]string
+		respType proto.Message
+		expected proto.Message
+	}{
+		{
+			"gRPC total account balance",
+			fmt.Sprintf("%s/cosmos/bank/v1beta1/balances/%s", baseURL, base64.URLEncoding.EncodeToString(val.Address)),
```

Comment:
Since the `address` path in the URL is defined as bytes in the proto files, and grpc-gateway decodes bytes with base64.URLEncoding, we (as in any HTTP client) need to encode the address as base64 URLEncoding first.

This has the unfortunate consequence of showing non-human-readable addresses in URLs:
```diff
+ /cosmos/bank/v1beta1/balances/u1-UOH-CHOvM2rNadSjtNARKkSk= // with base64
- /cosmos/bank/v1beta1/balances/cosmos1hd0egwrlsgwwhnx6kdd8228dxszy4yffnv2gtu // expected
```

What do people think here? Are you okay to use base64-encoded addresses in URLs?

If not, it's maybe worth to raise an issue on grpc-gateway to see if we can customize bytes decoding.